### ISSUE TITLE
Handlebars dependency has high-risk vulnerability

### ISSUE THREAD

**Issue by kevintechie** [amwmedia/node-plop#100]
Just got a notification that the handlebars dependency of node-plop has a high-risk vulnerability:

https://www.npmjs.com/advisories/755

**Comment by amwmedia** [comment]
as far as I can tell, `mixin-deep` is not a dependency of handlebars. It is in the package-lock.json for node-plop, but it's included in the `@types/globby` package (which is in devDependencies). 

Here's the result of `npm list mixin-deep`
```
node-plop@0.17.3 /Users/aworcester/github/node-plop
└─┬ @types/globby@8.0.0
  └─┬ fast-glob@2.2.4
    └─┬ micromatch@3.1.10
      └─┬ snapdragon@0.8.2
        └─┬ base@0.11.2
          └── mixin-deep@1.3.1
```

**Comment by amwmedia** [comment]
found it, fixed it. Thanks for bringing this to my attention, released with `v0.17.4`
